### PR TITLE
Add clippyx extension

### PR DESCRIPTION
**Pull request by 0ssamaak0**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/clippyx/src/caption.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+interface Image {
+  id: string;
+  path: string;
+}
+
+interface Preferences {
+  threshold: number;
+  topK: number;
+}
```
Comment (pernielsentikaer) [inline extensions/clippyx/src/caption.tsx:17]: You can remove this part, this is automaticlly generated in `raycast-env.d.ts`

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Is there a reason why all command has the same preferences, shouldn't it be enough to set for the extension itself?

- I'm not sure what the preferences it for, does the end user knows for sure?

- Maybe update the README with a link to the download page and tell a bit about it

- Could you record me a screencast of the extensions flow?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/clippyx/src/caption.tsx (hunk 1 above)
Comment (0ssamaak0) [reply extensions/clippyx/src/caption.tsx:17]: I got this error when I tried removing it

```
src/caption.tsx:27
:30 - error TS2339: Property 'threshold' does not exist on type 'ExtensionPreferences'.

27       threshold: preferences.threshold,
                                ~~~~~~~~~

src/caption.tsx:28:26 - error TS2339: Property 'topK' does not exist on type 'ExtensionPreferences'.

28       top_k: preferences.topK,
```

**Comment by 0ssamaak0** [comment]
> Hi 👋
> 
> Thanks for your contribution 💪
> 
> I have now tested your extension, and I have some feedback ready for you:
> 
> * Is there a reason why all command has the same preferences, shouldn't it be enough to set for the extension itself?
> * I'm not sure what the preferences it for, does the end user knows for sure?
> * Maybe update the README with a link to the download page and tell a bit about it
> * Could you record me a screencast of the extensions flow?
> 
> I'm looking forward to testing this extension again 🔥
> 
> Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.


- Yes, it's different since it uses different models / modalities
- The end user is already using CLIPPyX from the webui, but using it from raycast will be far more convinient
- I updated it and added link to the main repo
- here it is: 

https://github.com/user-attachments/assets/c1f1456a-0c4c-454a-ab36-3d5aafffcbaa

**Comment by pernielsentikaer** [comment]
Should be better now 🙂

**Comment by 0ssamaak0** [comment]
> Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension

Done ✅

Thank you for help

File: extensions/clippyx/package.json [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          "required": true,
+          "title": "Threshold",
+          "description": "Float value between 0 and 1",
+          "placeholder": "0.2",
+          "default": "0.2"
+        },
+        {
+          "name": "topK",
+          "type": "textfield",
+          "required": true,
```
Comment (pernielsentikaer) [inline extensions/clippyx/package.json:32]: Would it make sense to set all `required` to `false`  instead so the user don't need to set them before using each command?

Comment (0ssamaak0) [reply extensions/clippyx/package.json:32]: I think it's better to leave them `true`? they will be set in the first time only right?

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**